mongodb_depot.go

```diff
@@ -158,15 +158,15 @@ func (m *mongoDepot) CheckWithError(tag *depot.Tag) (bool, error) {
 func (m *mongoDepot) Get(tag *depot.Tag) ([]byte, error) {
 	name, key, err := getNameAndKey(tag)
 	if err != nil {
-		return nil, errors.Wrapf(err, "could not format name %s", name)
+		return nil, errors.Wrapf(err, "formatting name '%s'", name)
 	}
 
 	u := &User{}
 	if err = m.client.Database(m.databaseName).Collection(m.collectionName).FindOne(m.ctx, bson.D{{Key: userIDKey, Value: name}}).Decode(u); err != nil {
 		if err == mongo.ErrNoDocuments {
-			return nil, errors.Wrapf(err, "could not find %s in the database", name)
+			return nil, errors.Wrapf(err, "name '%s' not found", name)
 		}
-		return nil, errors.Wrapf(err, "problem looking up %s in the database", name)
+		return nil, errors.Wrapf(err, "looking up '%s' in the database", name)
```

Comment:
`"looking up name '%s' in the database"`

Reply:
Fixed.